scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/SortedBucketTransform.java

```diff
@@ -470,7 +475,7 @@ public void processElement(
               false,
               bucketId,
               effectiveParallelism,
-              context.getPipelineOptions());
+              pipelineOptions);
```

Comment:
any reason to change this if context is still required ?

Reply:
I will revert this